packages/byods/jest.config.js

```diff
@@ -0,0 +1,42 @@
+export default {
```

Comment:
There is a jest config present that we use for all the packages can we import that and then add our additional config on top of it? To use the newer tooling.
```
const config = require('@webex/jest-config-legacy');

module.exports = {...config, 
         ourConfig: 'ourValue' ;
};
```

Reply:
Addressed